lib/matplotlib/colorbar.py

```diff
@@ -1099,34 +1123,25 @@ def _reset_locator_formatter_scale(self):
         self.locator = None
         self.minorlocator = None
         self.formatter = None
+        longax = self._long_axis()
```

Comment:
`longax` isn't used below, so can just remove it? If you end up removing the `_set_scale()` function though, you may need to use this instead though.

Reply:
I'll keep the `_set_scale`, but you are right this was not being used.

Reply:
> `longax` wasn't removed.

Ooops, removed now.  I must have messed up a rebase at some point...